### PR TITLE
int32 import export

### PR DESCRIPTION
**Pull request by justin1121**

### CODE REVIEW

**Comment by justin1121** [comment]
@mortendahl this one is ready! CI coming next!

File: tf_big/cc/kernels/big_kernels.cc [hunk 1]
```diff
@@ -42,12 +44,13 @@ class BigImportOp : public OpKernel {
     OP_REQUIRES_OK(ctx, ctx->allocate_output(0, TensorShape{}, &val));
 
     BigTensor big;
-    big.FromTensor<string>(str);
+    big.FromTensor<T>(str);
```
Comment (mortendahl) [inline tf_big/cc/kernels/big_kernels.cc]: maybe rename `str` to something else now

File: tf_big/cc/kernels/big_kernels.cc [hunk 2]
```diff
@@ -60,7 +63,7 @@ class BigExportOp : public OpKernel {
     Tensor* str;
     OP_REQUIRES_OK(ctx, ctx->allocate_output(0, output_shape, &str));
 
-    val->ToTensor<string>(str);
+    val->ToTensor<T>(str);
```
Comment (mortendahl) [inline tf_big/cc/kernels/big_kernels.cc]: same: maybe rename `str`

File: tf_big/cc/kernels/big_tensor.cc [hunk 3]
```diff
@@ -2,7 +2,8 @@
 
 #include "big_tensor.h"
 
-BigTensor::BigTensor(MatrixXm mat) { value = mat; }
+namespace tfbig {
```
Comment (mortendahl) [inline tf_big/cc/kernels/big_tensor.cc]: `tf_big`?

File: tf_big/python/ops/big_ops_test.py [hunk 4]
```diff
@@ -13,10 +13,19 @@ def test_import_export(self):
     with tf.Session() as sess:
       inp = [[b"43424"]]
```
Comment (mortendahl) [inline tf_big/python/ops/big_ops_test.py]: note: we could probably get around the need for byte strings if we wrapped the ops in python land

File: tf_big/python/ops/big_ops_test.py [hunk 5]
```diff
@@ -13,10 +13,19 @@ def test_import_export(self):
...
       variant = big_import(inp)
-      output = big_export(variant)
+      output = big_export(variant, tf.string)
 
       assert sess.run(output) == inp
 
+  def test_export_int32(self):
+    with tf.Session() as sess:
+      inp = [[b"43424"]]
```
Comment (mortendahl) [inline tf_big/python/ops/big_ops_test.py]: can we import from int32 as well?

File: tf_big/python/ops/big_ops_test.py (hunk 4 above)
Comment (mortendahl) [reply tf_big/python/ops/big_ops_test.py]: we could also do some shape matching, and eg automatically convert a scalar into a (1,1) matrix

File: tf_big/cc/ops/big_ops.cc [hunk 6]
```diff
@@ -3,13 +3,15 @@
 
 // Input just a string for now, we make this more robust in the future
```
Comment (mortendahl) [inline tf_big/cc/ops/big_ops.cc]: comment outdated?

File: tf_big/cc/kernels/big_tensor.h [hunk 7]
```diff
@@ -42,12 +42,14 @@ struct NumTraits<mpz_class> : GenericNumTraits<mpz_class> {
 
 typedef Matrix<mpz_class, Dynamic, Dynamic> MatrixXm;
 
+namespace tfbig {
```
Comment (mortendahl) [inline tf_big/cc/kernels/big_tensor.h]: wondering if it makes sense to pull `BigTensor` code out of the kernels directory..?

Comment (justin1121) [reply tf_big/cc/kernels/big_tensor.h]: Yeah probably! https://github.com/tf-encrypted/tf-big/issues/15

**Comment by justin1121** [comment]
Fixed up all your comments! Merging now.

**Pull request merged**